run.py

```diff
@@ -0,0 +1,10 @@
+import cv2
+import numpy
+import ball_detection as bd
+
+img = cv2.imread("test_images/dis48_angle10.png")
```

Comment:
you could try and make this script take in an argument so you can pass in the filename. I like this package a lot: https://click.palletsprojects.com/en/8.0.x/ or there's also https://docs.python.org/3/library/argparse.html